var/spack/repos/builtin/packages/py-neuron-reduce/package.py

```diff
@@ -14,8 +14,9 @@ class PyNeuronReduce(PythonPackage):
     git = "git@github.com:orena1/neuron_reduce.git"
 
     version('develop', branch='master')
-    version('0.0.6', sha256='e75ce138ae049f550d72ec86c5cae4afcada577a2805d9dc50cbe80eff1f2256')
+    version('0.0.8', commit='3aada2ad3606723a6ebbaf39581153e38de24733')
```

Comment:
Tags?

Reply:
Unfortunately the author of the package didn't release. I have to use commits.

Reply:
OK